src/main/java/io/github/thebusybiscuit/slimefun4/api/events/AndroidMineEvent.java

```diff
@@ -24,18 +29,22 @@
 
     private final Block block;
     private final AndroidInstance android;
+    private final Collection<ItemStack> drops;
     private boolean cancelled;
 
     /**
      * @param block
      *            The mined {@link Block}
      * @param android
      *            The {@link AndroidInstance} that triggered this {@link Event}
+     * @param drops
+     *            The {@link List} of {@link ItemStack}s that will be dropped
```

Comment:
Inconsistency. You used `Collection` for the type here.
```suggestion
     *            The {@link Collection} of {@link ItemStack}s that will be dropped
```